src/app/components/MessageBanner/index.styles.tsx

```diff
@@ -0,0 +1,156 @@
+import { css, Theme } from '@emotion/react';
+import pixelsToRem from '../../utilities/pixelsToRem';
+
+const styles = {
+  container: () =>
```

Comment:
Believe we are missing the transparent border as per the AAC?

Reply:
There are lots of examples of this in the codebase: 

`border: 0.0625rem solid transparent;`

Reply:
Please could we size this 0.1875rem

Reply:
Sure thing! Thanks for checking 😄

Reply:
I've added this into the code, just have to check in the browser that it works correctly and then will push the change up.

Reply:
Awesome thanks!

Reply:
Change pushed and looks okay in the browser 😄